packages/flutter/test/material/outline_button_test.dart

```diff
@@ -0,0 +1,252 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' show SemanticsFlag;
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter/rendering.dart';
+
+import '../rendering/mock_canvas.dart';
+import '../widgets/semantics_tester.dart';
+
+void main() {
+  testWidgets('Outline button responds to tap when enabled', (WidgetTester tester) async {
+    int pressedCount = 0;
+
+    Widget buildFrame(VoidCallback onPressed) {
+      return new Directionality(
+        textDirection: TextDirection.ltr,
+        child: new Theme(
+          data: new ThemeData(),
+          child: new Center(
+            child: new OutlineButton(onPressed: onPressed),
+          ),
+        ),
+      );
+    }
+
+    await tester.pumpWidget(
+      buildFrame(() { pressedCount += 1; }),
+    );
+    expect(tester.widget<OutlineButton>(find.byType(OutlineButton)).enabled, true);
+    await tester.tap(find.byType(OutlineButton));
+    await tester.pumpAndSettle();
+    expect(pressedCount, 1);
+
+    await tester.pumpWidget(
+      buildFrame(null),
+    );
+    expect(tester.widget<OutlineButton>(find.byType(OutlineButton)).enabled, false);
+    await tester.tap(find.byType(OutlineButton));
```

Comment:
Finders are stateless. You can make `find.byType(OutlineButton)` a reusable named variable.

Reply:
OK, I introduced a variable.

Reply:
Hmm, I don't see it. Have you pushed all the changes?